common/lib/lebedev_laikov_f77.f

```diff
@@ -0,0 +1,6949 @@
+       subroutine gen_oh(code, num, x, y, z, w, a, b, v)
```

Comment:
Have a look in the DFTB+ version of the Lebedev–Laikov grids, the API changes slightly, but it helps you get rid of one of  “nightmare.”

https://github.com/dftbplus/dftbplus/blob/main/src/dftbp/extlibs/lebedev.F90

Reply:
Definitely, thanks for the information, I'll leave this to a follow-up PR to avoid breaking reproduction of references for now.